terraform/modules/cognito-app/outputs.tf

```diff
@@ -0,0 +1,5 @@
+output outputs {
+  value = {
+    app_clients = aws_cognito_user_pool_client.app_client
```

Comment:
Ah! This answers my earlier question, can we not name this app_client?

Reply:
Updated